### PR TITLE
XP-264 put coordinator as own package

### PR DESCRIPTION
**Pull request by wilk10**
### References

[XP-264](https://xainag.atlassian.net/browse/XP-264)

### Summary

The coordinator file inside the former `grpc` module contained different classes. This ticket breaks up that file and prepares the repo basic structure for cleaning up the legacy (local coordinator, benchmarks etc)

### Are there any open tasks/blockers for the ticket?

This ticket is the first to be completed for the story [XP-249 ](https://xainag.atlassian.net/browse/XP-249)

---

### Reviewer checklist

*Reviewer agreement:*

* Reviewers assign themselves at the start of the review.
* Reviewers do **not** commit or merge the merge request.
* Reviewers have to check and mark items in the checklist.

**Merge request checklist**

- [x] Conforms to the merge request title naming `XP-XXX <a description in imperative form>`.
- [x] Each commit conforms to the naming convention `XP-XXX <a description in imperative form>`.
- [x] Linked the ticket in the merge request title or the references section.
- [x] Added an informative merge request summary.

**Code checklist**

- [x] Conforms to the branch naming `XP-XXX-<a_small_stub>`.
- [x] Passed scope checks.
- [x] Added or updated tests if needed.
- [x] Added or updated code documentation if needed.
- [x] Conforms to Google docstring style.
- [x] Conforms to XAIN structlog style.

### CODE REVIEW

**Comment by wilk10** [comment]
thanks @PanicButtonPressed ! good catch, i fixed it as request

File: scripts/format.sh [hunk 1]
```diff
@@ -8,5 +8,5 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd $DIR/../
 
 isort --indent=4 -rc setup.py conftest.py benchmarks examples xain_fl docs/conf.py
-black --exclude "xain_fl/grpc/.*_pb2.*" setup.py conftest.py benchmarks examples xain_fl docs/conf.py
-clang-format -style="{Language: Proto, BasedOnStyle: Google}" -i  protobuf/xain_fl/grpc/*.proto
+black --exclude "xain_fl/cproto/.*_pb2.*" setup.py conftest.py benchmarks examples xain_fl docs/conf.py
+clang-format -style="{Language: Proto, BasedOnStyle: Google}" -i  protobuf/xain_fl/cproto/*.proto
```
Comment (dkravetz) [inline scripts/format.sh]: We can remove clang-format. It wasn't being used properly, and AFAIK, we're moving the proto files to a separate repo, meaning that the linting would have to happen somewhere else

Comment (dkravetz) [reply scripts/format.sh]: Also, please update the CI to reference the new directory:
```suggestion
~/.github/workflows/main.yml
```

Comment (wilk10) [reply scripts/format.sh]: thanks! removed clang_format and rebased with master to get the new CI dir

**Pull request merged**